README.md

```diff
@@ -31,23 +34,21 @@ console.log(isValid);
 // Output
 // true
 
-
 // Invalid path
 try {
-  validatePath()
+  validatePath();
 } catch (err) {
   console.log(err);
 
   // Output
-  /**
-   * {
-   *   status: 'err',
-   *   message: 'No path input',
-   *   data: {}
-   * }
-   **/
+	/**
```

Comment:
lets make these also 2 spaces

Reply:
this should be working with two spaces now, apologies as I don't know what my editor was trying to do with this spacing!